pkg/binding/format/doc.go

```diff
@@ -0,0 +1,8 @@
+package format
+
+/*
+Package format formats structured events.
+
+The "application/cloudevents+json" format is built-in and always
```

Comment:
this is only how http talks about this format, not the rest of the protocols.

Reply:
As far as I see from the spec, all protocol bindings which has both the structured and binary mode, uses `application/cloudevents+json` as content-type to define a cloudevent json encoded:

* https://github.com/cloudevents/spec/blob/master/amqp-protocol-binding.md#321-amqp-content-type
* https://github.com/cloudevents/spec/blob/master/amqp-protocol-binding.md#321-amqp-content-type
* https://github.com/cloudevents/spec/blob/master/kafka-protocol-binding.md#331-kafka-content-type
* https://github.com/cloudevents/spec/blob/master/mqtt-protocol-binding.md#321-mqtt-content-type